src/com/nutrons/FRamework/Robot.java

```diff
@@ -1,13 +1,39 @@
 package com.nutrons.FRamework;
 
 import edu.wpi.first.wpilibj.RobotBase;
+import io.reactivex.Flowable;
+
+import static com.nutrons.FRamework.CompMode.*;
 
 public class Robot extends RobotBase {
 
+    private Flowable<Boolean> enabled;
+    private Flowable<CompMode> mode;
+
+    public Robot() {
+        this.enabled = Util.changedValues(Util.toFlow(this::isEnabled));
+        this.mode = Util.changedValues(Flowable.merge(Util.toFlow(this::isAutonomous).filter((x) -> x).map((x) -> AUTO),
+                Util.toFlow(this::isOperatorControl).filter((x) -> x).map((x) -> TELEOP),
```

Comment:
The identity function is available as a static method in the java.util.Function class.